src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,16 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.PasswordAndRepeatPassword;
+
+@PasswordAndRepeatPassword
 public class UserRequestDto {
+    @NotNull
```

Comment:
Don’t repeat checks for fields: you must ensure email field not to be null in your custom Validator, so there is no need to put @NotNull for that field as well in Dto class. (checklist)